pkg/features/kube_features.go

```diff
@@ -62,6 +62,13 @@ const (
 	//
 	// Enables Kueue visibility on demand
 	VisibilityOnDemand featuregate.Feature = "VisibilityOnDemand"
+
+	// owner: @yaroslava-serdiuk
+	// kep: https://github.com/kubernetes-sigs/kueue/issues/1283
+	// alpha: v0.6
+	//
+	// Enable priority sorting within the cohort.
+	PrioritySortingWithinCohort featuregate.Feature = "PrioritySortingWithinCohort"
```

Comment:
@yaroslava-serdiuk @alculquicondor IIUC, even if clusterQueue doesn't belong to any cohort, once we enable this featuregate, priority would be ignored, right?

Reply:
If so, I'm wondering if we should name it `PrioritySorting`.

Reply:
the default behavior won't change, because this will be beta

Reply:
Ah, that's right. The scheduler will sort headsworkloads using the function `Less`. 
It means the sorting workloads by the function `Less` will apply only across cluster queues.

Thanks.